eks-anywhere-baremetal/Testers/Core/snapshot-tester/test-app/kustomization.yaml

```diff
@@ -1,5 +0,0 @@
-apiVersion: kustomize.config.k8s.io/v1beta1
-kind: Kustomization
-resources:
-  - claim.yaml
```

Comment:
Why is this file removed.

Reply:
Adding these two kustomization files back since we're using the v1beta1 API version that needs these elsewhere in the repository

Reply:
Per our discussion these resource pointer files are unnecessary